### PR TITLE
Feature/styleable ruler

### PR DESCRIPTION
**Pull request by frwg**

### CODE REVIEW

File: src/Mapbender/CoreBundle/Element/Type/RulerAdminType.php [hunk 1]
```diff
@@ -30,18 +31,40 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
-                'choices' => array(
-                    "line" => "line",
-                    "area" => "area",
-                ),
-                'choices_as_values' => true,
-            ))
+                    'required' => true,
+                    'choices' => array(
+                        "line" => "line",
+                        "area" => "area",
```
Comment (DeEgge) [inline src/Mapbender/CoreBundle/Element/Type/RulerAdminType.php:37]: `Line` and `Polygon` can be used as values... (see comment below)

File: src/Mapbender/CoreBundle/Resources/public/mapbender.element.ruler.js [hunk 2]
```diff
@@ -42,12 +42,33 @@
                 }
             }
 
+            var sketchSymbolizer = {};
+            var scheme = this.options.type === "line" ? "Line" : "Polygon";
```
Comment (DeEgge) [inline src/Mapbender/CoreBundle/Resources/public/mapbender.element.ruler.js:46]: ... this way we don't need the ternary operator.

File: src/Mapbender/CoreBundle/Element/Type/RulerAdminType.php [hunk 3]
```diff
@@ -30,18 +31,40 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
                 'label' => 'Immediate',
             ))
-        ;
+            ->add('strokeWidth', 'Symfony\Component\Form\Extension\Core\Type\IntegerType',
+                array('required' => false,
+                    'label' => 'Str. Width',
+                    'attr' => ['min' => 1, 'max' => 8],
+                    'data' => 1))
+            ->add('strokeOpacity', 'Symfony\Component\Form\Extension\Core\Type\RangeType',
+                array('required' => false, 'label' => 'Str. Opacity', 'attr' => [
```
Comment (DeEgge) [inline src/Mapbender/CoreBundle/Element/Type/RulerAdminType.php:51]: Please don't mix short and long `Array` syntax

Comment (frwg) [reply src/Mapbender/CoreBundle/Element/Type/RulerAdminType.php]: mxing of array syntax is resolved,  but decided against 'Line' and 'Polygon' as values in order to be minimally invasive.

**Comment by werrolf** [comment]
Can we please stop adding Openlayers 2 specific svg styling features in the middle of an engine transition phase?

**Pull request closed without merging**